servicecontrol/import-failed-messages.md

```diff
@@ -8,16 +8,49 @@ redirects:
+NOTE: Messages that have corrupt (i.e. unreadable, not deserializable) header data will fail to be process at all and be moved to the 'error' queue (by default `particular.servicecontrol.error` or `particular.servicecontrol.audit.error`)
 
- When a message that has failed the import is detected in the ServiceControl database, the [**Message Ingestion** custom check](/servicecontrol/servicecontrol-instances/#self-monitoring-via-custom-checks-failed-imports) is marked as failed to bring the failed import(s) to the administrator's attention.
+Messages that fail to be imported are stored in the ServiceControl database in the `FailedAuditImports` and `FailedErrorImports` collections.
 
-To reimport the failed messages, ServiceControl needs to be shut down and started from a command line with `--import-failed-errors` option. To import failed audit messages ServiceControl Audit needs to be shut down and started from a command line with `--import-failed-audits`. While in this mode, ServiceControl or ServiceControl Audit will not be processing its input queues. Once the imported failure message is re-processed successfully, the message will be available in ServiceInsight. 
+In addition, a log with the failure reason is written for the message in the [`%ServiceControl/LogPath%`](/servicecontrol/creating-config-file.md#host-settings-servicecontrollogpath)`\FailedImports\{Audit|Error}\%failureid%.txt`. These messages will not be visible in ServiceInsight.
+
+## Failed message custom check
+
+When a message that has failed the import is detected in the ServiceControl database, the [**Message Ingestion** custom check](/servicecontrol/servicecontrol-instances/#self-monitoring-via-custom-checks-failed-imports) is marked as failed to bring the failed import(s) to the administrator's attention.
+
+## How to reimport
+
+To reimport the failed messages, the instance to be shut down and started from a command line
+
+NOTE: The value to use for `--serviceName` is equal to the instance name. It also is available in the Windows Service information
+
+ServiceControl instance:
+
+Using the additional argument `--import-failed-errors`.
+
+```cmd
+ServiceControl.exe --serviceName=Particular.Servicecontrol --import-failed-errors
+```
+
+ServiceControl audit instance:
+
+Using the additional argument `--import-failed-audits`
+
+```cmd
+ServiceControl.Audit.exe --serviceName=Particular.Servicecontrol.Audit --import-failed-audits
+```
+
+While in import mode, ServiceControl or ServiceControl Audit will not be processing its input queues. Once the imported failure message is re-processed successfully, the message will be available in ServicePulse and ServiceInsight.
```

Comment:
```suggestion
While in import mode, ServiceControl or ServiceControl Audit will not be processing its input queues. Once the message is re-processed successfully, it will be available in ServicePulse and ServiceInsight. ServiceControl or ServiceControl Audit instance can then be started normally again.
```